packages/cli/src/config.rs

```diff
@@ -101,6 +107,7 @@ pub struct WebConfig {
     pub proxy: Option<Vec<WebProxyConfig>>,
     pub watcher: WebWatcherConfig,
     pub resource: WebResourceConfig,
+    pub https: WebHttpsConfig,
```

Comment:
Could this use [#[serde(default)]](https://serde.rs/attr-default.html) so that if no `[web.http]` section is found, it will use the default config? (This would make the PR a non-breaking change)

Reply:
Absolutely!